templates/sql-prunability-ec2/tests.yaml

```diff
@@ -0,0 +1,143 @@
+- name: Definitions
+  repetitions: -1
+  real_repetitions: &repetitions 1
+  parallel: &parallel False
+  duration: &duration 3m
+  clients: &clients 1
+  members: &members 5
```

Comment:
you use 5 members but 16 threads - the load will not be the same on each member which may make results harder to interpret.

Reply:
as discussed, the query is executed on random member unless partition aware client is used